numpy/core/shape_base.py

```diff
@@ -537,7 +599,7 @@ def block(arrays):
 
     Examples
     --------
-    The most common use of this function is to build a block matrix
+    The most common use of this function is to build a 2D block array
```

Comment:
Sorry for being unclear - we should keep the word matrix in our public docs for people googling "numpy block matrix" - but internally, we should be precise about what we mean when we say matrix. both this case and the one above were better as they were.

Reply:
forced push.